src/sentry/tagstore/types.py

```diff
@@ -38,7 +39,7 @@ def __setstate__(self, state):
 
 
 class TagKey(TagType):
-    __slots__ = ["key", "values_seen", "status"]
+    __slots__ = ("key", "values_seen", "status", "count", "top_values")
```

Comment:
this is a change in behaviour since these are now going to be considered for hashing and will be preserved across pickle boundaries -- arguably the original implementation was buggy because it did not preserve these